manzif/Banka
# #165485835 Add JWT authentication

#### What does this PR do?
Authenticates endpoints

#### Description of Task to be completed?
Generate tokens
Protect all transactions endpoints
Protect activate and deactivate accounts endpoints

##### How should this be manually tested?
`npm run server: start` to the run the app corresponding to this PR
If you check out the endpoints stated above you receive a message asking you to provide the appropriate token

#### What are the relevant pivotal tracker stories?
[#165485835](https://www.pivotaltracker.com/story/show/165485835)
